README.md

```diff
@@ -35,3 +43,27 @@ This command generates static content into the `build` directory and can be serv
 ### Deployment
 
 Deployment is fully automatic and thus requires no configuration or user interaction.
+
+
+## Contributing to Yoast Developer portal
+Howdy! It's really great that you want to contribute to the Yoast developer portal. The development of the portal takes place on GitHub.
+
+There are three main ways to contribute:
+1. [Contributing to issues](#contributing-to-issues)
+2. [Reviewing pull requests](#reviewing-pull-requests)
+3. [Contributing to code](#contributing-to-code)
+
+### Contributing to issues
+We use Jira to track changes and improvements to the portal that need to be discussed and completed. The portal related tickets are [in this board](#).
```

Comment:
Jira isn't open for people outside of the company, so I guess the description needs to just ask for a GitHub issue on this repository.